integrations/coinbase-integration/src/main/java/org/dash/wallet/integration/coinbase_integration/repository/CoinBaseRepository.kt

```diff
@@ -34,9 +35,12 @@ class CoinBaseRepository @Inject constructor(
     private val commitBuyOrderMapper: CommitBuyOrderMapper,
     private val coinbaseAddressMapper: CoinbaseAddressMapper
 ) : CoinBaseRepositoryInt {
-    override suspend fun getUserAccount() = safeApiCall {
+    private var userAccountInfo: List<CoinBaseUserAccountData> = listOf()
```

Comment:
All methods in this class call `/accounts` endpoint every time they need something which results in long processing times and loading dialogs. It's better to cache the result. `getUserAccount` will still update it periodically.
We might also want to add more sophisticated caching logic.